Thai/Travel/Travel.md

```diff
@@ -0,0 +1,7 @@
+﻿You will fall in love with Thailand. It is amazingly beautiful. We have plenty of stunning beaches and crystal clear water. Imagine you are at the sea now, what would you like to do?
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.